mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
is `2000` actually a named `FIFF` constant? Can you check? If so, we should add it to `mne.io.constants`